server/middlewares/build-page.js

```diff
@@ -42,7 +42,7 @@ function getCommonData(pagePath, i18n) {
     var pageName = path.basename(pagePath);
 
     return {
-        assetsPath: hosts.static + pagePath + '/_' + pageName,
+        assetsPath: hosts.static + pagePath + '/' + pageName,
```

Comment:
it shouldn't be hard to replace current mask with underscore with something like `*.{css,js}`, isn't it?